accumulo/src/main/java/org/vertexium/accumulo/AccumuloVertex.java

```diff
@@ -186,17 +184,13 @@ public static Vertex createFromIteratorValue(
 
     @Override
     public Iterable<Edge> getEdges(Direction direction, String label, FetchHints fetchHints, Authorizations authorizations) {
-        if (!getFetchHints().validateHasEdgeFetchHints(direction)) {
-            return null;
-        }
+        getFetchHints().validateHasEdgeFetchHints(direction);
```

Comment:
Seems like we need a new method `FetchHints.validateHasEdgeFetchHints(Direction, Label...)`

Reply:
done